src/lib/i18n/locales/fr-FR/translation.json

```diff
@@ -53,22 +53,22 @@
 	"Already have an account?": "Avez-vous déjà un compte ?",
 	"an assistant": "un assistant",
 	"and": "et",
-	"and {{COUNT}} more": "",
+	"and {{COUNT}} more": "et {{COUNT}} autres",
 	"and create a new shared link.": "et créer un nouveau lien partagé.",
 	"API Base URL": "URL de base de l'API",
 	"API Key": "Clé d'API",
 	"API Key created.": "Clé d'API générée.",
 	"API keys": "Clés d'API",
 	"April": "Avril",
-	"Archive": "Archivage",
+	"Archive": "Archiver",
```

Comment:
Some labels like this are actually verbs (button labels) and not nouns, so "Archiver" (verb) makes more sense than "Archivage" (noun).